docs/notes.md

```diff
@@ -0,0 +1,27 @@
+# Tracking the impact of changes
+
+Some changes in your organization, either internal or external to the environment, can impact your software delivery process. For example, changes in the team structure or membership, teams adopting new working processes, or major events affecting the system stability, might have a positive or negative impact on your teams' performance.
```

Comment:
I wonder if we may shorten this, merging the two sentences perhaps?

Reply:
Thank you for the proposal! 🙏  I was trying to shorten this paragraph, but as I want to keep the examples within the sentence (and not in a list), I prefer to keep two sentences, otherwise, it would be too long. Also, I want to pass the two ideas: the impact on the software delivery process, and the impact on the performance of the teams.